core/src/main/scala/fs2/io/ssh/Client.scala

```diff
@@ -17,65 +17,71 @@
 package fs2
 package io.ssh
 
-import cats.effect.{Blocker, Concurrent, ContextShift, Resource, Sync}
+import cats.effect.kernel.Async
```

Comment:
Prefer omitting `kernel` for consistency, generally
```suggestion
import cats.effect.Async
```

Reply:
👍